source/testing/test-helpers.md

```diff
@@ -37,7 +37,7 @@ next one starts.
     is complete.
 * `fillIn(selector, text)`
   - Fills in the selected input with the given text and returns a promise that
-     fulfills when all resulting async behavior is complete. Works with `<select>` elements as well as `<input>` elements.
+     fulfills when all resulting async behavior is complete. Works with `<select>` elements as well as `<input>` elements (with select elements, fill in the value of the option, e.g. 'true' and not the display, e.g. 'Yes').
```

Comment:
Sorry, poorly said. What I'm trying to say is that for `<input>`, there is no difference between the value and what's displayed on the screen, so it goes without saying that you'd use the value. With `<select>`, there is a difference, and so it is helpful to mention that you should use the value.

If we just said "Works with `<select>` elements as well as `<input>` elements (use the value of the option, e.g. 'true' and not the display, e.g. 'Yes').", the parenthetical comment doesn't make much sense in the context of the `<input>` tag. I'm struggling to come up with a general-case phrasing that doesn't feel confusing for `<input>`. Maybe you can suggest something?